scompose/project/project.py

```diff
@@ -529,10 +537,10 @@ def _create(
             create_func(
                 working_dir=self.working_dir,
                 writable_tmpfs=writable_tmpfs,
-                ip_address=lookup[instance.name],
+                ip_address=lookup[instance.get_replica_name()],
             )
 
-            created.append(instance.name)
+            created.add(instance.name)
```

Comment:
Same here - we aren't telling the user about replicas - just the parent name.

Reply:
Same rationale of the previous comment.